pwndbg/commands/ai.py

```diff
@@ -49,6 +71,16 @@ def set_dummy_mode(d=True):
     return
 
 
+def get_openai_api_key():
+    if config.ai_openai_api_key.value == "":
+        try:
+            config.ai_openai_api_key.value = os.environ["OPENAI_API_KEY"]
```

Comment:
Could do `api_key = os.environ.get("OPENAI_API_KEY", "")` and `if api_key: ...` instead